README.adoc

```diff
@@ -184,9 +161,9 @@ the `io_openliberty_guides_system_inMaintenance` property value to `true` in the
 `resources/CustomConfigSource.json` file, which turns the `system` service in maintenance.
 After saving the file, go back to your browser and
 refresh to the `http://localhost:9080/inventory/systems/localhost` URL to view the cached version of
-the properties. The `fallbackForGet()` method, which is the designated fallback method, is called
+the properties, which contain only the OS name and user name key and value pairs.
+The `fallbackForGet()` method, which is the designated fallback method, was called
```

Comment:
Say "is called" instead of "was called."

Reply:
fixed